### ISSUE TITLE
[Bug]: Chrome Throws lockdown errors

### ISSUE THREAD

**Issue by resilientboy** [MetaMask/metamask-extension#21113]
### Describe the bug

There are 2 errors thrown by chrome browser that goes away if i disable MetaMask.

### Expected behavior

No errors

### Screenshots

![Untitled](https://github.com/MetaMask/metamask-extension/assets/4083043/ce4e7502-4552-4118-a8a4-092a69c47082)

### Steps to reproduce

Opening any website on chrome while metamask is active as extention

### Error messages or log output

```shell
lockdown-run.js:17 Lockdown failed: TypeError: At intrinsics.Object.groupBy expected boolean not function
  at isAllowedPropertyValue (lockdown-install.js:1:53384)
  at isAllowedProperty (lockdown-install.js:1:53807)
  at visitProperties (lockdown-install.js:1:55095)
  at isAllowedPropertyValue (lockdown-install.js:1:53041)
  at isAllowedProperty (lockdown-install.js:1:53807)
  at visitProperties (lockdown-install.js:1:55095)
  at lockdown-install.js:1:55523
  at repairIntrinsics (lockdown-install.js:1:144597)
  at lockdown-install.js:1:145462
  at lockdown-run.js:4:3
(anonymous) @ lockdown-run.js:17
Show 1 more frame
Show less
lockdown-more.js:99 Protecting intrinsics failed: ReferenceError: harden is not defined
  at lockdown-more.js:69:13
  at Set.forEach (<anonymous>)
  at protectIntrinsics (lockdown-more.js:44:22)
  at lockdown-more.js:97:5
```

### Version

11.0.0

### Build type

None

### Browser

Chrome

### Operating system

Windows

**Comment by narasimha-1511** [comment]
This is due to the meta mask extensions

**Comment by anaamolnar** [comment]
Hello, @resilientboy. Thanks for reporting! We have a fix for this coming in version 11.1.0. I will close this in favor https://github.com/MetaMask/metamask-extension/issues/20847, as it is a duplicate. Thank you!
